src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,95 @@
+import usersFromServer from '../../api/users';
+import { Todo } from '../../types/Todo';
+import React, { FC, useState, FormEvent } from 'react';
+import { findUserById } from '../../utils/FindUserById';
+
+interface Props {
+  onAdd: (todo: Todo) => void;
+  todos: Todo[];
+}
+
+export const TodoForm: FC<Props> = ({ onAdd, todos }) => {
+  const [newTitle, setNewTitle] = useState('');
+  const [newUserId, setNewUserId] = useState(0);
+
+  const [titleError, setTitleError] = useState('');
+  const [userError, setUserError] = useState('');
```

Comment:
use `enum` for ur errors
```suggestion
  const [titleError, setTitleError] = useState<ErrorType>(ErrorType.DEFAULT);
  const [userError, setUserError] = useState<ErrorType>(ErrorType.DEFAULT);
```
```
export enum ErrorType {
  TITLE_DEFAULT: '',
  TITLE_ERROR: 'Your errors text',
  .........
}
```